plugin/plugin.go

```diff
@@ -133,12 +104,16 @@ func (e *Ratelimit) Evaluate(r rules.RuleMetadata, tx rules.TransactionState) {
 
 	if totalEventsOccuredInPreviousWindow < e.MaxEvents {
 		e.Zones[zone_name][currentTimeInSecond]++
-		fmt.Println(e.Zones)
+		log.Println(e.Zones)
 	} else {
 		// implement logic after ratelimit exceeded
-		fmt.Println("Ratelimit exceeded")
+		log.Println("Ratelimit exceeded")
```

Comment:
Better to use the `tx` logger as it includes all the context from the transaction.

Reply:
sure, I thought I would use tx.Logger in the end for all logs once other things are done.
Since now it seems to be almost done, I will fix all loggers to tx.Logger